inscriptions/ISic1508.xml

```diff
@@ -170,11 +175,18 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
+                <ab>
+                    <lb n="1"/>ὑπ<supplied reason="lost">ατείᾳ</supplied>
+                    <lb n="2"/>τῶν κυρίων <supplied reason="lost">ἡμῶν</supplied> <expan><abbr>Κωνστ</abbr><ex>αντίου</ex></expan> Σεβασ<supplied reason="lost">τ</supplied> <supplied reason="lost">τὸ</supplied> <gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost">καὶ</supplied> <gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
Formally the Sebast should be an unexpanded abbreviation I think, or just a word that can't be completed because of uncertainty about the inflection, but I wouldn't worry at this point